docs/release-notes/17.16.0 and 16.16.0.md

```diff
@@ -0,0 +1,9 @@
+# Model Builder 16.16.0 (VS2019) and 17.16.0 (VS2022) Release
+
+## New feature
+- Local Object Detection && Enhancing [Issue 2333](https://github.com/dotnet/machinelearning-modelbuilder/issues/2333), [Issue 2610](https://github.com/dotnet/machinelearning-modelbuilder/issues/2610)
+
+## Bug fixing
+- Single float column being set to string [Issue 2478](https://github.com/dotnet/machinelearning-modelbuilder/issues/2478)
+- Tensorflow runtime fail to install when there's space in user name folder [Issue 2621](https://github.com/dotnet/machinelearning-modelbuilder/issues/2621)
+- Label missing | Data path blank [Issue 2623](https://github.com/dotnet/machinelearning-modelbuilder/issues/2623)
```

Comment:
This is not an existing bug  it is  because we added  coco data set type, it is the part of the big work of add local OB